intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -2851,12 +2981,14 @@ def scan_exchange_identifier(field):
     ('Honeypot-Amplification-DDoS-Events', 'event4_honeypot_ddos_amp', honeypot_ddos_amp),
     ('Honeypot-Brute-Force-Events', 'event4_honeypot_brute_force', honeypot_brute_force),
     ('Honeypot-Darknet', 'event4_honeypot_darknet', event4_honeypot_darknet),
+    ('Honeypot_http_scan', 'event4_honeypot_http_scan', event4_honeypot_http_scan),
     ('ICS-Scanners', 'hp_ics_scan', ics_scanners),
     ('IPv6-Sinkhole-HTTP-Drone', 'sinkhole6_http', ipv6_sinkhole_http_drone),  # legacy (replaced by event46_sinkhole_http)
     ('IP-Spoofer-Events', 'event4_ip_spoofer', event4_ip_spoofer),
     ('Microsoft-Sinkhole', 'microsoft_sinkhole', microsoft_sinkhole),  # legacy (replaced by event46_sinkhole_http)
     ('Microsoft-Sinkhole-Events IPv4', 'event4_microsoft_sinkhole', event46_sinkhole),
     ('Microsoft-Sinkhole-Events-HTTP IPv4', 'event4_microsoft_sinkhole_http', event46_sinkhole_http),
+    ('Microsoft_sinkhole_http', 'event4_microsoft_sinkhole_http', event4_microsoft_sinkhole_http),
```

Comment:
As in https://github.com/certtools/intelmq/pull/2050#discussion_r692443726 I don't understand this change.

Reply:
a need feed was added by shadow server with that name, if the document doesn't have that, the parser bot stops

Reply:
a new feed*

Reply:
The shadowserver page does not know a feed with that name, just https://www.shadowserver.org/what-we-do/network-reporting/microsoft-sinkhole-http-events-report/ (Microsoft Sinkhole HTTP Events), which we already have in our mapping.